### ISSUE TITLE
[Requested Feature] Cool down on paper balls

### ISSUE THREAD

**Issue by FelipeCanales94** [ConnorMulqueen/Geesenado#106]
Paper balls are too overpowered since you can just spam them. Maybe have a cool down on it after 15 throws? Or have them come in drops of 10 so every time you pick up the paper ball drop, you get 10 paper balls instead of an unlimited amount

**Comment by jaredgav** [comment]
Yes the weapon feature set can use some tweaking. We plan to re-assign weapon power and damage to make the game more even. Thank you for this! We'll implement this soon!
